pkgs/development/libraries/amtk/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl
+, pkgconfig, gnome3 }:
+let
+  version = "4.99.1";
+  pname = "amtk";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "00fhvw5y638z584s8cfdslh47ngfzcgx4f0b0456sw8p754j3f8d";
+  };
+
+  passthru.updateScript = gnome3.updateScript { packageName = pname; };
+
+  nativeBuildInputs = [
+    gnome3.gtk.dev
+    pkgconfig
+  ];
+
+  doCheck = false;
+  # TODO: seems like tests need a display:
```

Comment:
Maybe running those in xvfb would help: https://github.com/NixOS/nixpkgs/blob/f7397f21ab841955eb277919f19035cbc336e88b/pkgs/development/libraries/gtksourceview/4.x.nix#L36

Reply:
seems to fail with this:
```
FAIL: test-file-metadata
========================

/file/get_set_metadata: OK
/file/load_save_metadata_sync:
(./test-file-metadata:20931): Tepl-WARNING **: 14:41:36.942: GVfs metadata is not supported. Fallback to TeplMetadataManager. Either GVfs is not correctly installed or GVfs metadata are not supported on this platform. In the latter case, you should configure Tepl with --disable-gvfs-metadata.
FAIL test-file-metadata (exit status: 133)
```

I'll try to find a solution

Reply:
`amtk` tests work with this trick though, so that's also nice

Reply:
We are building on tmpfs so some file attributes might not work. Probably nothing we can do about it.